index.d.ts

```diff
@@ -388,3 +388,12 @@ export const enum SvgExportFlag {
 }
 
 export function convertSVGTextToPath(svg: Buffer | string): Buffer
+
+export interface LoadImageOptions {
+  maxRedirects?: number
```

Comment:
I'd like to add `headers` option to `LoadImageOptions ` in case somebody want to fetch images with secrets token